### DISCUSSION TITLE
no me anda el compilador

### DISCUSSION THREAD

**Discussion by jazlagos** [INGCOM-UNRN-P1/cursada-2023#20] [Preguntas y respuestas] [answered]
escribo `gcc --version` en la consola y me aparece "gcc" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
hice todos los pasos de wiki.

**Comment by martinvilu** [comment, marked as answer]
Si escribiendo en la consola no funciona como `C:\mingw64\bin\gcc.exe`, entonces hay un problema en los primeros pasos, en el descompactar y copiar los archivos a la ubicación correcta.
